bin/cloudgov/create-service-account

```diff
@@ -1,18 +1,31 @@
-#!/bin/bash
+#!/bin/env bash
 
```

Comment:
Please, let's add some documentation while we're here. Either comments here, or a usage statement as we have in some other scripts. Something like: 

Creates a service account in the specified space (which must be current) and assigns that account roles in all spaces. 
Usage: bin/cloudgov/create-service-account space [keyname] [--reset-key]

Keyname defaults to "cci". Service account will be named keyname-service-account. 
If --reset-key is specified, the existing service account will be deleted and replaced.

Reply:
On it - thank you for looking!